modules/@amperka/quaddisplay2.js

```diff
@@ -87,6 +89,7 @@ QuadDisplay.prototype.frame = function(shift) {
     this._shift = this._data.length;
   }
   this._spi.send(this._data.slice(this._shift, this._shift + 4), this._cs);
+  digitalWrite(this._cs,0);
```

Comment:
`digitalWrite(this._cs,1);` has lost above. Is it OK?

Reply:
Yes, It's work without `digitalWrite(this._cs,1); `